desc/objectives/objective_funs.py

```diff
@@ -565,10 +566,19 @@ def jit(self):
 
     def _check_dimensions(self):
         """Check that len(target) = len(weight) = dim_f."""
-        self._target = np.asarray(self._target)
+        if isinstance(self.target, tuple):
+            self._target = tuple([np.asarray(tar) for tar in self._target])
+            for tar in self.target:
+                if not is_broadcastable((self.dim_f,), tar.shape):
```

Comment:
this is for if we have some vector of target like in FixBoundaryR, and we passed in a target R_b that is actually a tuple of (R_b_lower,R_b_upper)?

Reply:
Correct. Although these target bounds aren't intended for linear objectives that get used as constraints like `FixBoundaryR`. A better example would be for the `ForceBalance` objective, where you are giving (F_lower, F_upper).